compiler/noirc_frontend/src/hir/comptime/interpreter/builtin.rs

```diff
@@ -307,6 +312,31 @@ fn slice_insert(
     Ok(Value::Slice(values, typ))
 }
 
+// fn as_module(quoted: Quoted) -> Option<Module>
+fn quoted_as_module(
+    interpreter: &mut Interpreter,
+    arguments: Vec<(Value, Location)>,
+    return_type: Type,
+    location: Location,
+) -> IResult<Value> {
+    let argument = check_one_argument(arguments, location)?;
+
+    let tokens = get_quoted(argument, location)?;
+    let quoted = add_token_spans(tokens.clone(), location.span);
+
+    let option_value = parser::path_no_turbofish()
+        .parse(quoted)
+        .ok()
+        .and_then(|path| {
+            interpreter.elaborate_item(interpreter.current_function, |elaborator| {
+                elaborator.resolve_module_by_path(path)
+            })
+        })
+        .map(Value::ModuleDefinition);
```

Comment:
```suggestion
    let path = parser::path_no_turbofish().parse(quoted).ok();
    let option_value = path.and_then(|path| {
        let module = interpreter.elaborate_item(interpreter.current_function, |elaborator| {
            elaborator.resolve_module_by_path(path)
        });
        Value::ModuleDefinition(module)
    });
```
Trying to fix the formatting here a bit. Short lines like `.ok()` really bother me.